bindings/node.js/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "include": ["*"],
```

Comment:
As matter of general education. Does `"*"` imply "don't pay attention to irrelevant files"? In other words is `"*"` effectively equal to `"*.ts"`? And in this specific context, would it be sufficient to include just "runnable.ts"?

Reply:
`"*"` tells the Typescript compiler to consider all local files for compilation. In normal projects, one usually targets `src` which contains an `index.ts` file.